### PR TITLE
Convert all CLI build targets to MSBuild Tasks.

### PR DESCRIPTION
**Pull request by eerhardt**
@brthor @livarcocc @piotrpMSFT @sokket

### CODE REVIEW

**Comment by eerhardt** [comment]
@brthor @livarcocc @piotrpMSFT @sokket - I'm out tomorrow, but you guys are all collaborators on my fork.  So if you want, you can udpate this PR to fix anything that is broken and merge it as you guys feel necessary.

File: NuGet.Config [hunk 1]
```diff
@@ -6,5 +6,6 @@
     <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="myget.org dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
```
Comment (eerhardt) [inline NuGet.Config]: @brthor - is this still needed?  I wasn't sure.  I needed it for the MSBuild nuget packages I was using in order to make our classes inherit from ITask.

**Comment by eerhardt** [comment]
Test CentOS7.1 x64 Debug Build

**Comment by sokket** [comment]
LGTM

**Pull request merged**